its/src/test/java/com/sonar/it/csharp/AnalysisWarningsTest.java

```diff
@@ -37,8 +37,8 @@ public class AnalysisWarningsTest {
   @Rule
   public TemporaryFolder temp = TestUtils.createTempFolder();
 
-  @Before
-  public void init() {
+  @BeforeClass
+  public static void init() {
```

Comment:
These changes are pretty irrelevant, but there is no reason to run the `server is not null` check per test, since it should be started once in total anyways, so `BeforeClass` is semantically more correct IMO.